app/tests/spec/views/sign_up.js

```diff
@@ -872,6 +873,53 @@ function (chai, $, sinon, p, View, Coppa, Session, AuthErrors, Metrics,
           });
       });
     });
+
+    describe('passwordCheck', function () {
+      beforeEach(function () {
+        view.passwordcheck = new PasswordCheck();
```

Comment:
A `passwordcheck` instance should already be created by the `render` function, I don't believe this is needed.